### PR TITLE
tigmint 1.2.1

### PR DESCRIPTION
**Pull request by lcoombe**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by lcoombe** [comment]
Failure on linux looks unrelated to tigmint:
```
==> brew style brewsci/bio
==> FAILED
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/biobloomtools.rb:3:9: C: Description shouldn't start with the formula name.
  desc "BioBloom Tools (BBT): Bloom filter for bioinformatics"
        ^^^^^^^^^^^^^^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/cap3.rb:3:9: C: Description shouldn't start with the formula name.
  desc "CAP3: A DNA Sequence Assembly Program"
        ^^^^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/humann2.rb:3:9: C: Description shouldn't start with the formula name.
  desc "HUMAnN2: The HMP Unified Metabolic Analysis Network"
        ^^^^^^^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/magic-blast.rb:2:9: C: Description shouldn't start with the formula name.
  desc "Magic BLAST read mapper"
        ^^^^^^^^^^^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/metaphlan.rb:3:9: C: Description shouldn't start with the formula name.
  desc "MetaPhlAn: Metagenomic Phylogenetic Analysis"
        ^^^^^^^^^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/quast.rb:6:9: C: Description shouldn't start with the formula name.
  desc "QUAST: Quality Assessment Tool for Genome Assemblies"
        ^^^^^
/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/sortmerna.rb:3:9: C: Description shouldn't start with the formula name.
  desc "SortMeRNA: filter metatranscriptomic ribosomal RNA"
        ^^^^^^^^^
```

File: Formula/tigmint.rb [hunk 1]
```diff
@@ -16,23 +15,16 @@ class Tigmint < Formula
   end
 
   depends_on "bedtools"
-  depends_on "python"
-
-  resource "cython" do
-    url "https://files.pythonhosted.org/packages/21/89/ca320e5b45d381ae0df74c4b5694f1471c1b2453c5eb4bac3449f5970481/Cython-0.28.5.tar.gz"
-    sha256 "b64575241f64f6ec005a4d4137339fb0ba5e156e826db2fdb5f458060d9979e0"
-  end
+  depends_on "minimap2"
```
Comment (sjackman) [inline Formula/tigmint.rb:18]:
```suggestion
  depends_on "brewsci/bio/minimap2"
```

File: Formula/tigmint.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
 class Tigmint < Formula
   # cite Jackman_2018: "https://doi.org/10.1186/s12859-018-2425-6"
-  desc "Correct misassemblies using linked reads"
+  desc "Correct misassemblies using linked or long reads"
```
Comment (sjackman) [inline Formula/tigmint.rb:3]: 🎉 ❤️

**Comment by sjackman** [comment]
The `brew style brewsci/bio` issue has been fixed. Could you please rebase this PR onto `origin/develop`?

**Comment by lcoombe** [comment]
Just rebased on `develop` and that fixed the issues! 😄

**Comment by sjackman** [comment]
Merged. Thanks, Lauren!

**Pull request merged**